.github/workflows/auto-publish.yml

```
@@ -1,72 +1,71 @@
-          node-version: 20 # 使用Node.js 20版本
-          registry-url: 'https://registry.npmjs.org' # 设置npm registry地址
+          node-version: 20
 
-      # 步骤4: 获取pnpm缓存目录路径
       - name: Get pnpm store directory
         id: pnpm-cache
         run: |
           echo "pnpm_cache_dir=$(pnpm store path)" >> $GITHUB_OUTPUT
 
-      # 步骤5: 配置pnpm缓存
       - uses: actions/cache@v3
```

actionlint (1.7.12)

[error] the runner of "actions/cache@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)